docs/upgrading/graphcommerce-6-1-to-6-2.md

```diff
@@ -87,3 +87,67 @@ In your package.json
     "@graphql-mesh/json-schema": "latest",
     "@graphql-mesh/transform-filter-schema": "latest",
 ```
+
+## Upgrading your Hygraph schema
+
+In version 6.2 and onwards it is possible to upgrade your Hygraph schema using
+the Graphcommerce Hygraph CLI. This is a tool that automatically modifes your
+Hygraph schema without the need of adding models/components/enumarations
+manually. The following steps are needed to utilize this tool:
+
+1. Install the @graphcommerce/hygraph-cli package
+
+2. Create a Hygraph migration token
```

Comment:
I think it might be best to create a separate Hygraph documentation page, something like `docs/hygraph/README.md` where we write about hygraph where the only documentation right now is how to use the CLI.

These upgrade instructions should specify which migrations are expected to be ran:

-  Use the [Hygraph CLI] to apply migrations x and y.